yt_dlp/extractor/npo.py

```diff
@@ -291,188 +319,8 @@ def _get_info(self, url, video_id):
 
         return info
 
-    def _get_old_info(self, video_id):
```

Comment:
You are sure this is no longer used by any video?

Reply:
The used domains `e.omroep.nl` and `ida.omroep.nl` no longer resolve